client/document/index.jsx

```diff
@@ -100,16 +103,28 @@ class Document extends Component {
 
 		const isRTL = isLocaleRtl( lang );
 
+		const redirectTo = typeof query?.redirect_to === 'string' ? query.redirect_to.split( '?' ) : [];
+
+		// Get the client ID from the redirect URL to cover the case of a login URL without the "client_id" parameter.
+		// e.g. /log-in/link/use
+		const clientId = new URLSearchParams( redirectTo[ 1 ] ).get( 'client_id' );
```

Comment:
Previously, we utilized the [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL) to obtain the client_id. This API requires the input to be in HTTP URL format; otherwise, it will throw a JavaScript fatal error. However, the query.redirect_to of the VideoPress's signup page does not conform to this format.

Now, we have adopted the use of [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams), which boasts high error tolerance, thereby ensuring that the page will not break during Server Side Rendering (SSR).